packages/bridge-ui/src/domain/bridge.ts

```diff
@@ -39,6 +39,7 @@ interface Bridge {
   RequiresAllowance(opts: ApproveOpts): Promise<boolean>;
   Approve(opts: ApproveOpts): Promise<Transaction>;
   Bridge(opts: BridgeOpts): Promise<Transaction>;
+  EstimateGas(opts: BridgeOpts): Promise<BigNumber>;
```

Comment:
@cyberhorsey Instead adding a bridge option I added this method so that even in the case of ERC20 we can check if the user has enough ETH balance for gas.

Reply:
okie!